README.md

```diff
@@ -67,7 +67,10 @@ See the [documentation](https://mcstatus.readthedocs.io) to find what you can do
 
 ### Command Line Interface
 
-This only works with Java servers; Bedrock is not yet supported. Use `mcstatus -h` to see helpful information on how to use this script.
+The mcstatus library includes a simple CLI. Once installed, it can be used through:
+```bash
+python3 -m mcstatus --help
```

Comment:
I suggest leaving this as `mcstatus --help`. Beginners often don't know about placing the correct python3 binary on their path so that module loading works appropriately. Some python environment managers only expose python3.X when a python3 binary is already present.

Reply:
It does indeed match. Carry on, while I go hide in shame.
![homer sinking into bushes](https://media0.giphy.com/media/v1.Y2lkPTc5MGI3NjExYTI1cmNzNW1veGJuMm4xb3poeDZpa2NtODg1cjUyYjE5cmdsNGx2eCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/COYGe9rZvfiaQ/giphy.webp)